src/Illuminate/Foundation/Testing/DatabaseTransactionsManager.php

```diff
@@ -6,6 +6,21 @@
 
 class DatabaseTransactionsManager extends BaseManager
 {
+    /**
+     * Number of connections transacting on tests to be skiped and run the callbacks correctly.
+     */
+    protected int $connectionsTransacting = 1;
```

Comment:
Why not just pass in the array of connections? This makes it easier if we need to utilise the connection name in the future.

Reply:
Ah, yeah, makes sense, will update it later.

Reply:
Done.